app/assets/javascripts/expand.js

```diff
@@ -46,7 +46,7 @@ $(document).ready(function() {
       //para5.appendChild(complete_node);
 
       var title_node = document.createTextNode(title);
-      // title_header.appendChild(title_node);
+      title_header.appendChild(title_node);
```

Comment:
Identifier 'title_header' is not in camel case.<br>Identifier 'title_node' is not in camel case.